pom.xml

```diff
@@ -61,6 +61,14 @@
             </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.19.1</version>
+                <configuration>
+                    <runOrder>random</runOrder>
+                </configuration>
+            </plugin>
```

Comment:
a couple of little benefits here, fixed surefire version so it wont vary by maven distro and random test run order to try and catch problems with test independence